easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-intel-2022b.eb

```diff
@@ -0,0 +1,50 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'Simple-DFTD3'
+version = '0.7.0'
+
+homepage = 'https://dftd3.readthedocs.io'
+description = """Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to
+provide a user-friendly and uniform interface to the D3 dispersion model and
+for the calculation of DFT-D3 dispersion corrections."""
+
+toolchain = {'name': 'intel', 'version': '2022b'}
+# In EasyBuild 4.8.2 is a bug in the Intel toolchain, so we need to turn off OpenMP here and add
+# -fopenmp instead as Bart Oldman suggested
+toolchainopts = {'usempi': False, 'openmp': False, 'pic': True, 'extra_fflags': '-fopenmp'}
```

Comment:
Since both usempi and openmp are False by default you can just drop them from toolchainopts when setting is False

Reply:
I have removed the `'usempi': False` bit as suggested. However, the `'openmp': False` was actually wrong, as in the `configure` step we are using `-DWITH_OpenMP=TRUE`. So I have set that to `true` as well. I have also removed the `-fopenmp` and the related comment from the Intel toolchain EasyConfig file.